model_info/viwiki.articletopic.md

```diff
@@ -551,411 +551,411 @@ Model Information:
+		Geography.Regions.Africa.Western Africa      0.965
+		Geography.Regions.Americas.Central America   0.916
+		Geography.Regions.Americas.North America     0.925
+		Geography.Regions.Americas.South America     0.935
+		Geography.Regions.Asia.Asia*                 0.914
+		Geography.Regions.Asia.Central Asia          0.953
+		Geography.Regions.Asia.East Asia             0.945
+		Geography.Regions.Asia.North Asia            0.97
+		Geography.Regions.Asia.South Asia            0.922
+		Geography.Regions.Asia.Southeast Asia        0.906
+		Geography.Regions.Asia.West Asia             0.949
+		Geography.Regions.Europe.Eastern Europe      0.95
+		Geography.Regions.Europe.Europe*             0.924
+		Geography.Regions.Europe.Northern Europe     0.913
+		Geography.Regions.Europe.Southern Europe     0.935
+		Geography.Regions.Europe.Western Europe      0.942
+		Geography.Regions.Oceania                    0.93
 		History and Society.Business and economics   0.972
-		History and Society.Education                0.98
-		History and Society.History                  0.961
-		History and Society.Military and warfare     0.98
-		History and Society.Politics and government  0.961
-		History and Society.Society                  0.946
-		History and Society.Transportation           0.99
-		STEM.Biology                                 0.987
-		STEM.Chemistry                               0.988
+		History and Society.Education                0.972
+		History and Society.History                  0.951
+		History and Society.Military and warfare     0.974
+		History and Society.Politics and government  0.926
+		History and Society.Society                  0.902
+		History and Society.Transportation           0.987
+		STEM.Biology                                 0.986
+		STEM.Chemistry                               0.987
 		STEM.Computing                               0.987
-		STEM.Earth and environment                   0.98
-		STEM.Engineering                             0.986
-		STEM.Libraries & Information                 0.974
-		STEM.Mathematics                             0.989
-		STEM.Medicine & Health                       0.981
-		STEM.Physics                                 0.988
-		STEM.STEM*                                   0.983
-		STEM.Space                                   0.994
-		STEM.Technology                              0.981
+		STEM.Earth and environment                   0.975
+		STEM.Engineering                             0.983
+		STEM.Libraries & Information                 0.961
+		STEM.Mathematics                             0.985
+		STEM.Medicine & Health                       0.974
+		STEM.Physics                                 0.986
+		STEM.STEM*                                   0.981
+		STEM.Space                                   0.989
+		STEM.Technology                              0.975
 		-------------------------------------------  -----
-	pr_auc (micro=0.826, macro=0.684):
+	pr_auc (micro=0.711, macro=0.533):
```

Comment:
Non-minor loss in fitness.  Might need to iterate on this one.

Reply:
What do you mean by 'iterate' in this case? Train more?

Reply:
Woops.  Meant "iterate".  We might need to increase the vocab size or some other hyperparameter for Vietnamese.